arkouda/pdarrayclass.py

```diff
@@ -396,63 +470,79 @@ def __setitem__(self, key, value):
             raise TypeError("Unhandled key type: {} ({})".\
                             format(key, type(key)))
 
-    def fill(self, value : object) -> None:
+    def fill(self, value : 'pdarray') -> None:
         """
         Fill the array (in place) with a constant value.
         """
         generic_msg("set {} {} {}".format(self.name, 
                                         self.dtype.name, self.format_other(value)))
 
-    def any(self):
+    def any(self) -> bool:
         """
         Return True iff any element of the array evaluates to True.
         """
         return any(self)
 
-    def all(self):
+    def all(self) -> bool:
         """
         Return True iff all elements of the array evaluate to True.
         """
         return all(self)
 
-    def is_sorted(self):
+    def is_sorted(self) -> bool:
         """
         Return True iff the array is monotonically non-decreasing.
+        
+        Parameters
+        ----------
+        None
+        
+        Returns
+        -------
+        bool 
+            Indicates if the array is monotonically non-decreasing
+            
+        Raises
+        ------
+        TypeError
+            Raised if pda is not a pdarray instance
+        RuntimeError
+            Raised if there's a server-side error thrown
         """
         return is_sorted(self)
 
-    def sum(self):
+    def sum(self) -> float:
```

Comment:
`sum` could return a np.floar64 or np.int64 depending on `dtype` of input `pdarray`

Reply:
fixed!